src/libraries/Microsoft.PowerFx.Interpreter/Functions/Library.cs

```diff
@@ -1383,6 +1383,17 @@ static Library()
                     returnBehavior: ReturnBehavior.AlwaysEvaluateAndReturnResult,
                     targetFunction: RoundDown)
             },
+            {
+                BuiltinFunctionsCore.Search,
+                StandardErrorHandlingAsync<FormulaValue>(
+                    BuiltinFunctionsCore.Search.Name,
+                    expandArguments: NoArgExpansion,
+                    replaceBlankValues: NoOpAlreadyHandledByIR,
+                    checkRuntimeTypes: ShowColumnsTypeChecker,

```

Comment:
>ShowColumnsTypeChecker [](http://example.com/codeflow?start=39&length=22)

The ShowColumnsTypeChecker will allow a record in the first argument, which we don't allow in Search.

Reply:
Makes sense, Thanks for pointing it out.